app/src/main/kotlin/org/fossify/musicplayer/activities/QueueActivity.kt

```diff
@@ -115,7 +116,9 @@ class QueueActivity : SimpleControllerActivity() {
     }
 
     private fun onSearchQueryChanged(text: String) {
-        val filtered = tracksIgnoringSearch.filter { it.title.contains(text, true) }.toMutableList() as ArrayList<Track>
+        val normalizedText = text.normalizeString()
+        val filtered = tracksIgnoringSearch.filter {
+            it.title.normalizeString().contains(normalizedText, true) }.toMutableList() as ArrayList<Track>
```

Comment:
There's unformatted code (lambda) everywhere now. Please respect the guidelines and review your changes.

```suggestion
        val filtered = tracksIgnoringSearch.filter {
            it.title.normalizeString().contains(normalizedText, true)
        }.toMutableList() as ArrayList<Track>```